Strings/Problem24/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Text;
+
+namespace Problem24
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            /*
+            Write a program that reads a string from the console and replaces every
+            sequence of identical letters in it with a single letter (the repeating
+            letter). Example: "aaaaabbbbbcdddeeeedssaa" -> "abcdedsa".
+            */
+
+            string input = Console.ReadLine();
+
+            StringBuilder stringBuilder = new StringBuilder();
+
+            stringBuilder.Append(input[0]);
```

Comment:
Can be done in a function.
why stringBuilder.Append(input[0]); outside of the cycle?